builtin/credential/okta/backend.go

```diff
@@ -191,8 +196,9 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username, pas
 			}
 		}
 
-		// Okta push and totp are currently supported. If a totp passcode is provided during
-		// login and is supported, that will be the preferred method.
+		// Okta push and totp, and Google totp are currently supported.
+		// If a totp passcode is provided during login and is supported,
+		// that will be the preferred method.
 		switch {
```

Comment:
This switch case also handles push factor, but the Google provider [doesn't support that](https://developer.okta.com/docs/reference/api/factors/#supported-factors-for-providers). Should we update this switch so that push factor can only be set if the provider is Okta?

Reply:
Sure, I can add a check for the provider to only use push factor for Okta.